app.py

```diff
@@ -82,30 +82,40 @@ def skill():
     return jsonify({})
 
 
+def check_user_data(api_data):
+    """
+    Check if data is valid
+    """
+    if api_data is not None:
+        name = api_data.get("name")
+        phone = api_data.get("phone")
+        email = api_data.get("email")
+
+    return name, phone, email  # pylint: disable=R1710
+
 @app.route("/resume/user", methods=["GET", "POST"])
-def user():
+def user(): # pylint: disable=R1710
```

Comment:
Can we actually fix this instead of disabling the linting? You can Google the problem, I believe you need to always return something (currently you don't have a return statement when the method is not GET or POST).